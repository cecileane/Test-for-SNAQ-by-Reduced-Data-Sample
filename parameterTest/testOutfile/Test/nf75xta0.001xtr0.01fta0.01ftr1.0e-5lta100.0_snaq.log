optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=0.01,
                       xtolAbs=0.001, xtolRel=0.01.
 max number of failed proposals = 75, liktolAbs = 100.0.
rootname for files: nf75xta0.001xtr0.01fta0.01ftr1.0e-5lta100.0_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 8142
seed: 8142 for run 1
Sat Oct  1 17:40:24 2016

 BEGIN SNaQ for run 1, seed 8142 and hmax 1
Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 123 (failures: 75) with -loglik=330.94203 and ht_min=[0.13808,0.0,0.66453,10.0,0.09277]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 8		 7		 1	 |	 NA 		 NA 		 NA 
mvorigin	 25		 25		 0	 |	 0		 0		 --
mvtarget	 25		 25		 0	 |	 0		 0		 --
chdir		 4		 4		 0	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 61		 33		 0	 |	 0		 0		 --
Total		 123		 94		 1	 |	 0		 0		 0
Proportion	 -- 		 1.0		 0.0	 |	 -- 		 0.0		 0.0

 FINISHED SNaQ for run 1, -loglik of best 330.9420330370395
(2,3,((4,(5,(6,#H7:::0.13807662634708523):10.0):0.6645264844750609):0.09276855594916494,(1)#H7:::0.8619233736529148):0.0);
---------------------
seed: 82555 for run 2
Sat Oct  1 17:40:28 2016

 BEGIN SNaQ for run 2, seed 82555 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 86 (failures: 75) with -loglik=300.47585 and ht_min=[0.49184,0.0,0.0,0.17212,8.84599]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 5		 5		 1	 |	 NA 		 NA 		 NA 
mvorigin	 24		 24		 0	 |	 0		 0		 --
mvtarget	 25		 25		 0	 |	 0		 0		 --
chdir		 4		 4		 0	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 28		 22		 0	 |	 0		 0		 --
Total		 86		 80		 1	 |	 0		 0		 0
Proportion	 -- 		 1.0		 0.0	 |	 -- 		 0.0		 0.0

 FINISHED SNaQ for run 2, -loglik of best 300.47584737862496
(3,((5,4):0.0,(6,(1,(2)#H7:::0.5081584384009283):8.84599255919938):0.17212279862491134):0.0,#H7:::0.49184156159907166);
---------------------

Sat Oct  1 17:40:31 2016
MaxNet is (3,((5,4):0.0,(6,(1,(2)#H7:::0.5081584384009283):8.84599255919938):0.17212279862491134):0.0,#H7:::0.49184156159907166); 
with -loglik 300.47584737862496
