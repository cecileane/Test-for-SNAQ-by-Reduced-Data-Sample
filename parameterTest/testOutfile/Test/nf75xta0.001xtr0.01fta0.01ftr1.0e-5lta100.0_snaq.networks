(3,((5,4):0.0,(6,(1,(2)#H7:::0.5081584384009283):8.84599255919938):0.17212279862491134):0.0,#H7:::0.49184156159907166);, with -loglik 300.47584737862496 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
(3,((5,4):0.0,(6,#H7:::0.3547884025253134):0.9090246381687235):0.06226822120084327,(2,(1)#H7:::0.6452115974746866):2.127683505902838);, with -loglik 283.09055115364725
(3,((5,4):0.0,(6)#H7:::0.8732136878692134):0.061305134981098144,(2,(1,#H7:::0.12678631213078667):3.2079255000070175):0.8146403014047675);, with -loglik 297.9225386287966
((3)#H7:::0.9488997604266313,(5,4):0.0,(6,(1,(2,#H7:::0.05110023957336873):0.0):0.7254004360293976):0.015160638481755489);, with -loglik 311.92946837446414
(3,((5,4):0.0)#H7:0.0::0.7011754034677642,(2,(1,(#H7:0.08301051478744152::0.2988245965322358,6):8.709162217673228):0.0):0.49222991231004637);, with -loglik 352.9044340116407
