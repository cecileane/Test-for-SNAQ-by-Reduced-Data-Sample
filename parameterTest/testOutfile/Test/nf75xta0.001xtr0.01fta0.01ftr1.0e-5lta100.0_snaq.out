(3,((5,4):0.0,(6,(1,(2)#H7:::0.5081584384009283):8.84599255919938):0.17212279862491134):0.0,#H7:::0.49184156159907166); -Ploglik = 300.47584737862496
 Dendroscope: (3,((5,4):0.0,(6,(1,(2)#H7):8.84599255919938):0.17212279862491134):0.0,#H7);
 Elapsed time: 6.9367573 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 (3,((5,4):0.0,(6,(1,(2)#H7:::0.5081584384009283):8.84599255919938):0.17212279862491134):0.0,#H7:::0.49184156159907166);, with -loglik 300.47584737862496
 (2,3,((4,(5,(6,#H7:::0.13807662634708523):10.0):0.6645264844750609):0.09276855594916494,(1)#H7:::0.8619233736529148):0.0);, with -loglik 330.9420330370395
-------
